### PR TITLE
simplify wording for `mc replicate add` and default sane values

### PR DESCRIPTION
**Pull request by harshavardhana**

### CODE REVIEW

File: cmd/replicate-add.go [hunk 1]
```diff
@@ -83,27 +84,29 @@ FLAGS:
  {{range .VisibleFlags}}{{.}}
  {{end}}
 EXAMPLES:
- 1. Add replication configuration rule on bucket "mybucket" for alias "myminio" to replicate all objects with tags
+ 1. Add replication configuration rule on bucket "mybucket" for alias "myminio" to replicate all operations in an active-active replication setup.
```
Comment (poornas) [inline cmd/replicate-add.go:87]:
```suggestion
 1. Add replication configuration rule on bucket "mybucket" for alias "myminio" to replicate all operations for prefix "prefix" in an active-active replication setup.
```

File: cmd/replicate-add.go [hunk 2]
```diff
@@ -83,27 +84,29 @@ FLAGS:
...
- 3. Add replication configuration rule with existing object replication, delete marker replication and versioned deletes
- 	 enabled on bucket "mybucket" for alias "myminio".
-	 {{.Prompt}} {{.HelpName}} myminio/mybucket/prefix --tags "key1=value1&key2=value2" \
-		--storage-class "STANDARD"  \
-		--remote-bucket 'arn:minio:replica::c5be6b16-769d-432a-9ef1-4567081f3566:destbucket' \
-		--priority 1 \
-		--replicate "existing-objects,delete,delete-marker"
+    {{.Prompt}} {{.HelpName}} myminio/mybucket/prefix --remote-bucket "arn:minio:replica::c5be6b16-769d-432a-9ef1-4567081f3566:destbucket" \
+        --tags "key1=value1&key2=value2" \
+        --replicate "delete,delete-marker" \
```
Comment (poornas) [inline cmd/replicate-add.go:95]: delete-marker and delete replication is disallowed when a rule has tags

**Review by poornas — CHANGES_REQUESTED** [review]

File: cmd/replicate-add.go [hunk 3]
```diff
@@ -62,7 +62,8 @@ var replicateAddFlags = []cli.Flag{
 	},
 	cli.StringFlag{
 		Name:  "replicate",
-		Usage: "comma separated list to enable replication of delete markers, deletion of versioned objects and replica metadata sync(in the case of active-active replication).Valid options are \"delete-marker\", \"delete\" ,\"replica-metadata-sync\", \"existing-objects\" and \"\"",
+		Value: `delete-marker,delete,existing-objects,metadata-sync`,
+		Usage: `comma separated list to enable replication of soft deletes, permanent deletes, existing objects and metadata sync`,
```
Comment (poornas) [inline cmd/replicate-add.go:66]: this usage description change is needed in `mc replicate edit` also

Comment (harshavardhana) [reply cmd/replicate-add.go:66]: will send that in separate PR @poornas

**Review by poornas — APPROVED** [review]

**Pull request merged**